### PR TITLE
fix: makes precache update expiration before domain table

### PR DESCRIPTION
**Pull request by jurajpiar**
adds id to expiration table definition

As I had a different version of the ui-config that was generated by the dev project, I was missing the auction registrar address and thus did not capture the error in my last PR.
This is to fix the issue with foreign key constraints when writing to domains table.

### CODE REVIEW

**Comment by julianmrodri** [comment]
@jurajpiar  I tried this and get Invalid Date in the UI for all domains. is this related? Maybe the Expired date is not being stored properly? 
image

File: src/services/rns/models/expiration.model.ts [hunk 1]
```diff
@@ -3,6 +3,9 @@ import { Column, DataType, Model, Table } from 'sequelize-typescript'
 @Table({ freezeTableName: true, tableName: 'rns_domain_expiration', timestamps: false })
 export default class DomainExpiration extends Model {
   @Column({ primaryKey: true, type: DataType.STRING })
+  id!: string
+
+  @Column({ type: DataType.STRING })
   tokenId!: string
```
Comment (vojtechsimetka) [inline src/services/rns/models/expiration.model.ts:9]: Do I understand it right, that you are duplicating the `tokenId`? This seems like quite lazy solution.

File: src/services/rns/hooks/domain.hooks.ts [hunk 2]
```diff
@@ -74,8 +82,9 @@ export default {
               }
             }
           }
-          discardQuery('placed')
-          discardQuery('name')
+          // discardQuery('placed')
```
Comment (AuHau) [inline src/services/rns/hooks/domain.hooks.ts:85]: Remove this, please :-)

**Comment by julianmrodri** [comment]
Worked fine. Lets correct the suggestions and get this merged!

File: src/services/rns/hooks/domain.hooks.ts (hunk 2 above)
Comment (jurajpiar) [reply src/services/rns/hooks/domain.hooks.ts:85]: oh yea thanks

**Pull request merged**